packages/console/src/pages/SignInExperience/tabs/SignUpAndSignIn/components/SocialConnectorEditBox/index.tsx

```diff
@@ -20,8 +21,12 @@ type Props = {
 function SocialConnectorEditBox({ value, onChange }: Props) {
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
   const { data: connectorData, error } = useConnectorGroups();
+  const socialConnectors = useMemo(
+    () => connectorData?.filter(({ type }) => type === ConnectorType.Social),
+    [connectorData]
+  );
 
-  if (!connectorData || error) {
+  if (!socialConnectors || error) {
     return null;
   }
```

Comment:
Looks like a logic change? Is this expected?

Reply:
According to the code, if any non-social connector exist, e.g emailConenctor, should not return null.

Reply:
updated.